bookwyrm/views/preferences/export.py

```diff
@@ -70,6 +71,16 @@ def post(self, request):
 
             book.rating = review_rating.rating if review_rating else None
 
+            readthrough = (
+                models.ReadThrough.objects.filter(user=request.user, book=book)
+                .order_by("-finish_date")
+                .first()
+            )
+            if readthrough:
```

Comment:
readthrough dates are `DateTime`s, which unfortunately means all the dates come through with a rather unhelpful timestamp like `2024-01-03 00:00:00+00:00`. You might want to consider converting these to dates like so:
```py
if readthrough:
    book.start_date = readthrough.start_date.date() if readthrough.start_date else ""
    book.finish_date = readthrough.finish_date.date() if readthrough.finish_date else ""
    book.stopped_date = readthrough.stopped_date.date() if readthrough.stopped_date else ""
```

Reply:
Great point! Thanks for pointing the solution, too.

I'm wondering: why is the if statement needed?  wouldn't `.date()` convert to date any DateTime anyway?

Reply:
I see that the if statement is needed (otherwise, I generate an error, but still not sure why is this needed)

Reply:
It's annoying syntax but the reason is that at least one of these will always be `None` (but we don't know which) and `None` doesn't have a `date()` function.

Reply:
actually it probably should be `else None`.

Reply:
All of this makes sense. I've added the `else None` in my last commit